designs/virtualization/4.7/snapshots/snapshots.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Virtualization
+version: 4.7
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VMs list](img/0-0.jpg)
+From the virtual machines menu item, the user selects “Create Snapshot” from a VM item’s kebab dropdown.
+The same action can be taken from a VM details Action dropdown.
+
+![Virtualization - turn off VM notification](img/0-1.jpg)
+In case the VM is running, the user will be asked to turn it off to make a snapshot
+
+![Virtualization - Create VM snapshot](img/1-0.jpg)
+The user enters the Create snapshot screen.
+This screen includes:
+- Snapshot name field
+- Dropdown to selecting Snapshot Class
+
+The snapshot class is auto-selected to the class matching with the disks' pvc's storage class.
+The default selection is set to the same snapshot class default when taking a snapshot for PVC on the Storage>PVC>Snapshot screen.
+In case no clear storage class is the default, this dropdown will be set to --select--.
+
+### Excluded disks
+
+![Virtualization - "un-snapshotable" disks](img/1-0-b.jpg)
```

Comment:
I talked with Micheal Henrikson / @aglitke further about this and I'm not sure we'll be able to list the disks like this. We may just have to surface a static message about the types of volumes that can be included. Can you follow up with Michael and confirm?